python/ray/rllib/test/test_local.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from ray.rllib.agents.ppo import PPOAgent, DEFAULT_CONFIG
+import ray
+
+
+class LocalModeTest(unittest.TestCase):
+    def testLocal(self):
+        ray.init(local_mode=True)
+        cf = DEFAULT_CONFIG.copy()
+        agent = PPOAgent(cf, "CartPole-v0")
+        print(agent.train())
```

Comment:
Doesn't Jenkins run in 2.7...?

Reply:
What's wrong?

Reply:
Oh the print() statement should fail, but you got the fix.